src/components/ChatRoom.tsx

```diff
@@ -112,7 +131,17 @@ function Editor({ roomId, message, onClose, ...rest }: {
     <Textarea value={markdown} onChange={e => setMarkdown(e.target.value)}
       autoFocus background="white" height={200} {...rest}
     />
+
     <HStack>
+      <Select placeholder="选择模版" 
+        onChange={(e) => insertSnippet(snippets.find(snippet => 
+          snippet.title === e.target.value)?.text || "")} 
+        width="180px" marginRight={10} variant="brand">
```

Comment:
what does it look like if we don't specify width, marginRight, variant and just use default style?